docs/iris/gallery_code/meteorology/plot_lagged_ensemble.py

```diff
@@ -116,42 +112,29 @@ def main():
 
     # set a global title for the postage stamps with the date formated by
     # "monthname year"
-    plt.suptitle(
-        "Surface temperature ensemble forecasts for %s"
-        % (time.strftime("%B %Y"),)
-    )
+    title = "Surface temperature ensemble forecasts for {}"
+    plt.suptitle(title.format(time.strftime("%B %Y")))
```

Comment:
@rcomer Even better, use an f-string here? i.e.,
```python
plt.suptitle(f"Surface temperature ensemble forecasts for {time.strftime('%B %Y')}")
```

Reply:
Ooooo I've never used an f-string before 😁

Reply:
Just as `format` is a step-up from `%`, f-strings are a step-up from `format` IMHO

e.g., generally once you've tried them, you don't look back. I kinda love'em now.